AdobeStockAdminUi/Test/Mftf/ActionGroup/AssertAdminFailedAuthenticationMessageActionGroup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminFailedAuthenticationMessageActionGroup">
+        <arguments>
+            <argument name="message" type="string" defaultValue="Failed to authenticate to Adobe Stock API. Please correct the API credentials"/>
```

Comment:
Can we use a data entity to store this text? As I know data should be separated from the logic.
Maybe this file can be used: https://github.com/magento/adobe-stock-integration/blob/develop/AdobeStockImageAdminUi/Test/Mftf/Data/AdobeStockModalData.xml

Reply:
Agree with this one 👍